### PR TITLE
Update @blockly/dev-scripts package

### PR DESCRIPTION
**Pull request by samelhusseini**
Add a README for the @blockly/dev-scripts package.
Fix up package.json

### CODE REVIEW

File: packages/dev/scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# @blockly/dev-scripts [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
+
+This package includes scripts and configuration used by Blockly extension
```
Comment (alschmiedt) [inline packages/dev/scripts/README.md]: configuration -> configurations

Comment (alschmiedt) [inline packages/dev/scripts/README.md]: maybe just choose between Blockly extensions or Blockly packages?

**Review by alschmiedt — APPROVED** [review]

File: packages/dev/scripts/README.md (hunk 1 above)
Comment (samelhusseini) [reply packages/dev/scripts/README.md]: Good point.

Comment (samelhusseini) [reply packages/dev/scripts/README.md]: Actually I agree with the standardization, but I actually think we should call them: 
``extension packages. ``

Comment (alschmiedt) [reply packages/dev/scripts/README.md]: Ok, any reason? I'm fine either way, just thinking that we have everything under packages and don't really seem to use extension anywhere else.

**Review by alschmiedt — APPROVED** [review]

File: packages/dev/scripts/README.md (hunk 1 above)
Comment (samelhusseini) [reply packages/dev/scripts/README.md]: Blockly extension, could be confused core Blockly extensions: https://developers.google.com/blockly/guides/create-custom-blocks/extensions

Maybe ``Blockly package`` ain't too bad. They are npm packages afterall.
Under packages, we're referring to reach item by its type (plugin, field, block).

Comment (samelhusseini) [reply packages/dev/scripts/README.md]: Problem with Blockly package is that it doesn't really tell me anything about it being an extension on top of Blockly, hence why I landed on ``extension packages``

Comment (samelhusseini) [reply packages/dev/scripts/README.md]: I'm open to ideas.

**Comment by samelhusseini** [comment]
Will hold off on merging this until we come to a consensus about naming.

**Comment by samelhusseini** [comment]
Closing in favour of https://github.com/google/blockly-samples/pull/207

**Pull request closed without merging**